sources/packages/web/src/views/aest/student/applicationDetails/ApplicationRestrictionsManagement.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <full-page-container layout-template="centered">
+    <template #header>
+      <header-navigator
+        title="Student Applications"
+        :routeLocation="{
+          name: AESTRoutesConst.STUDENT_APPLICATIONS,
+          params: { studentId },
+        }"
+        subTitle="Application Restriction Management"
+      />
+      <div v-if="applicationDetail.applicationNumber">
+        <span class="label-bold-normal">Application #: </span>
+        <span
+          >{{ emptyStringFiller(applicationDetail.applicationNumber) }} |
+        </span>
+        <span class="label-bold-normal">Institution: </span>
+        <span>{{ applicationDetail.applicationInstitutionName }} | </span>
+        <span class="label-bold-normal">Study Dates: </span>
+        <span
+          >{{ dateOnlyLongString(applicationDetail.applicationStartDate) }}
+          -
+          {{ dateOnlyLongString(applicationDetail.applicationEndDate) }}
+          |
+        </span>
+        <span class="label-bold-normal">Type: </span>
+        <span
+          >{{
+            getOfferingIntensity(applicationDetail.applicationOfferingIntensity)
+          }}
+        </span>
+      </div>
+    </template>
+    <history-bypassed-restrictions
+      class="mb-5"
+      :applicationId="applicationId"
+      :applicationStatus="applicationDetail.applicationStatus"
+      :key="historyKey"
+    />
+    <div class="text-center mt-6">
+      <span class="header-text-small">Date submitted: </span
+      ><span class="value-text-small">{{
```

Comment:
I see that the classes here are from the `formio-shared.scss`. Not sure if it is a good idea to use them here. I think the same can be created in `base.scss` and used from there. @dheepak-aot please confirm.

Reply:
This component is removed.